dali/test/python/test_operator_normalize.py

```diff
@@ -390,10 +390,13 @@ def _test_up_to_5D_all_axis_combinations(device):
                     yield _run_test, device, batch_size, dim, None, axis_names, batch_norm
 
 def test_cpu_up_to_5D_all_axis_combinations():
-    for device in ["cpu", "gpu"]:
+    for device in ["gpu"]:
```

Comment:
Don't get it, test is called `*_cpu`. Run GPU and CPU, right now runs only GPU.

Reply:
Nice catch! It was a temporary change for debugging.